websites/K/Komiku/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+    clientId: "868085258371870820"
+  }),
+
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+// Presence On
+presence.on("UpdateData", async () => {
+  const buttons = await presence.getSetting("buttons"),
+    time = await presence.getSetting("timestamps"),
+    presenceData: PresenceData = {
+      details: "Page not Supported", // If the page cannot be recognized
+      largeImageKey: "logo"
+    };
```

Comment:
```suggestion
    presenceData: PresenceData = {
      details: "Page not Supported", // If the page cannot be recognized
      largeImageKey: "logo",
      startTimestamp: browsingStamp
    };
```

And remove the other startTimestamps